test/addons/symlinked-module/test.js

```diff
@@ -30,5 +30,5 @@ const sub = require('./submodule');
   const mod = require(path.join(i, 'binding.node'));
   assert.notStrictEqual(mod, null);
   assert.strictEqual(mod.hello(), 'world');
-  assert.doesNotThrow(() => sub.test(i));
+  sub.test(i);
```

Comment:
In a couple of places (like here) the `doesNotThrow()` is really a comment in disguise.

Someone reading the new code might be confused and think the `sub.test(i)` is a bug or dead code because the return value isn't checked, not realizing it's there to check a side effect (throwing an exception.)

I would suggest rewriting lines like these to `sub.test(i);  // Should not throw.`.  Apart from that getting rid of `doesNotThrow()` is (IMO) a good move.

Reply:
I went throw the code and added comments were I felt it made sense. I also some times improved a few tests along the way and added punctuation and caps and things like that.